### PR TITLE
Add redeem popup

### PR DESCRIPTION
**Pull request by sarveshpro**
add redeem popup
image

### CODE REVIEW

**Comment by sagararyal** [comment]
@sarveshpro @harshvitra this should show up only if tenant = planet or ttc

File: public/static/locales/de/common.json [hunk 1]
```diff
@@ -46,6 +46,8 @@
   "continueToLogin": "Weiter zum Login",
   "supportsUNEP": "Unterstützt das UNEP",
   "plantedBy": "Gepflanzt von",
-  "mapInfo": "Esri Info",
-  "aboutUs": "Über uns"
-}
\ No newline at end of file
+  "mapInfo": "Esri",
+  "aboutUs": "Über uns",
+  "redeemPopup": "Baumgeschenk Bekommen? Log dich ein um den Code Einzulösen",
```
Comment (norbertschuler) [inline public/static/locales/de/common.json:51]: This need to be fixed (in LingoHu)b: "Baumgeschenk bekommen? Log dich ein, um den Code einzulösen!".

**Comment by norbertschuler** [comment]
> @sarveshpro @harshvitra this should show up only if tenant = planet or ttc

Added an optional flag to the tenants config to mark the tenants which should show the hint (I do not want to add any code like `if (config.tenantName === 'ttc')` to our codebase. We should avoid that!

**Review by norbertschuler — APPROVED** [review]
Works for me. Do we really want this to be a hot fix to production? Otherwise we should rebase the PR on develop and merge into the next normal release.

**Comment by norbertschuler** [comment]
```
Failed to compile.
--
11:51:37.520 | ModuleNotFoundError: Module not found: Error: Can't resolve '../../../../../i18n/server' in '/vercel/workpath0/src/features/common/Layout/RedeemPopup'

```

**Pull request merged**